packages/react-icons/src/utils/SvgIcon.scss

```diff
@@ -0,0 +1,13 @@
+.root {
+  display: inline-block;
+  vertical-align: middle;
+  speak: none;
+  width: 1em;
+  height: 1em;
```

Comment:
```
line-height: 1; 
```
Also unsure that width/height should be set. Why not just 16px if we're going to have a default?

Reply:
setting w & h with `1em` allows user to control the icon side using `font-size`.